subsystems/motor/SensorMotor.java

```diff
@@ -11,59 +11,76 @@
 public abstract class SensorMotor extends Motor {
 	protected final MotionController motionController;
 	private boolean isMotionControlEnabled;
-
-	public SensorMotor(String name, boolean inverted, SpeedModifier speedModifier, MotionController motionController,
-		SpeedController... motors) {
+	
+	public SensorMotor(String name, boolean inverted, SpeedModifier speedModifier, MotionController motionController, SpeedController... motors) {
```

Comment:
Please update your formatter before merging.
The split in this line is correct under the new paradigm of _avoid sideways scrolling_